src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,69 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Lotto {
+    private static final int NUMBER_COUNT = 6;
+    private static final int MIN_NUMBER = 1;
+    private static final int MAX_NUMBER = 45;
+
+    protected List<Integer> numbers;
+
+    public Lotto() {
+        List<Integer> randomNumbers = getRandomNumbers();
+        numbers = sorted(randomNumbers);
+
+    }
+
+    private List<Integer> getRandomNumbers() {
+        List<Integer> availableNumbers = new ArrayList<>();
+
+        for (int i = MIN_NUMBER; i <= MAX_NUMBER; i++) {
+            availableNumbers.add(i);
+        }
+
+        Collections.shuffle(availableNumbers);
+        return availableNumbers.stream()
+                .limit(6)
```

Comment:
이미 매직넘버 추출을 잘 해놓으셨으니 활용하면 더 좋겠습니다 🙏 
```suggestion
                .limit(NUMBER_COUNT)
```